pkgs/applications/networking/cluster/coop-cloud/abra.nix

```diff
@@ -0,0 +1,26 @@
+{buildGoModule, fetchFromGitea, lib}:
+buildGoModule rec {
+  pname = "abra";
+  version = "0.4.1-alpha";
+  src = fetchFromGitea {
+    domain = "git.coopcloud.tech";
+    owner = "coop-cloud";
+    repo = "abra";
+    rev = "4ba15df9b702fdeec4cc2089a826da7ef69fd711";
```

Comment:
`rev = "refs/tags/${version}";` maybe?

Reply:
The main issue is that the magic package auto-update bot won't be able to automatically update this package, since just bumping the version won't be enough. It looks like the git commit sha is optional, looking at the program source, and it's only being used in `--help` text and such. Unless you feel strongly, I have a weak preference for just dropping the hash and making the package maintenance easier and automatable.

Reply:
Alright, I removed it. For some reason this particular gitea instance returns internal server error for `"refs/tags/${version}"`, but works for just `"${version}"`. I will raise this with the maintainers, but in the meanwhile maybe this is ok?

Reply:
Actually, it looks like all other uses for `fetchFromGitea` use just the tag as `rev` and not github style `refs/tags/${tag}`. So, I think i will not raise this anywhere, maybe gitea just works like this.